bsp/bsp_timer.cpp

```diff
@@ -119,7 +119,9 @@ void setFreq( uint8_t pwm, uint16_t freq )
         p.tc->COUNT16.CTRLA.bit.ENABLE = 0;
         syncTC_16(p.tc);
         p.tc->COUNT16.CTRLA.bit.PRESCALER = prescale;
-        enableTimer(p.tc);
+        if (freq > 0){
+          enableTimer(p.tc);
+        }
 #ifdef USE_TCC_TIMERS
     }
     else{
```

Comment:
I don’t know, I don’t have a SAMD21 device to test the behavior, that’s why I didn’t change it.

If you think that it should be changed too I can update the code there, otherwise I would prefer to leave it to someone who can actually test it.